internal/service/folders/data_source.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/datasource"
 	"github.com/hashicorp/terraform-plugin-framework/types"
 
-	// "github.com/hashicorp/terraform-plugin-log/tflog"
+	// "github.com/hashicorp/terraform-plugin-log/tflog".
```

Comment:
@rek what is the . for?

Reply:
```sh
$ golangci-lint run 

internal/service/folders/data_source.go:10:54: Comment should end in a period (godot)
	// "github.com/hashicorp/terraform-plugin-log/tflog"
	                                                    ^
```

It fixes this issue. But perhaps I should just remove that line instead.